server/sonar-server/src/main/java/org/sonar/server/exceptions/Message.java

```diff
@@ -20,13 +20,15 @@
 package org.sonar.server.exceptions;
 
 import com.google.common.base.MoreObjects;
-import org.apache.commons.lang.StringUtils;
-
+import java.util.Arrays;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
+import org.apache.commons.lang.StringUtils;
 
-import java.util.Arrays;
-
+/**
+ * @deprecated Bundle key should be replaced by real message as only english is returned on server side messages
+ */
+@Deprecated
 public class Message {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)